apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -63,7 +63,13 @@
 
 #ifdef CONFIG_FS_SMARTFS
 
-#if defined(CONFIG_SIDK_S5JT200_AUTOMOUNT_USERFS)
+#if defined(CONFIG_SMARTFS_MULTI_ROOT_DIRS) && defined(CONFIG_ARTIK05X_AUTOMOUNT_USERFS)
```

Comment:
This is not good. If someone changes a dev path on config, this will make an error. Because we provide a config to set dev path, this should work with config.
Let's redefine it as shown below:
```
#define VFS_FILE_PATH MOUNT_DIR"vfs"
```

Reply:
In your commit, above my suggestion is enough. Last sentence is not for this commit.